lib/web_ui/lib/src/engine/text_editing/text_capitalization.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of engine;
+
+/// Controls the capitalization of the text.
+///
+/// This corresponds to Flutter's [TextCapitalization].
+///
+/// Uses `text-transform` css property.
+/// See: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
+enum TextCapitalization {
+  /// Uppercase for the first letter of each word.
+  words,
+
+  /// Currently not implemented on Flutter Web. Uppercase for the first letter
+  /// of each sentence.
+  sentences,
+
+  /// Uppercase for each letter.
+  characters,
+
+  /// Lowercase for each letter.
+  none,
+}
+
+/// Helper class for text capitalization.
+///
+/// Uses `text-transform` css property.
+/// See: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
+class TextCapitalizationConfig {
+  final TextCapitalization textCapitalization;
+
+  static final RegExp wordExp = new RegExp(r"(\w+)");
+  static final RegExp whiteSpaceExp = new RegExp(r"(\s+)");
```

Comment:
Not necessary anymore :)

Reply:
yess :) I forgot to remove all those things. Too much distraction from license tests. thanks for the review.

Reply:
done.